tests/phpunit/Persistence/SitelinkBasedStatementsLookupTest.php

```diff
@@ -41,18 +41,18 @@ protected function setUp(): void {
 		);
 	}
 
-	public function testPageWithoutSitelinkReturnsNoStatements(): void {
-		$this->assertPageHasStatements( $this->createPage(), [] );
```

Comment:
Besides the same naming issue, this assertion method is hiding the Act part of the test. I didn't bother refactoring it now, but looks like it would have been better if this code remained inlined